### PR TITLE
Spanish translation

### PR DESCRIPTION
**Pull request by SrGeneroso**
Hi, I've translated the readme file, I hope it helps. I'll be testing the project to see what could I do with it. Thanks for including vue!

### CODE REVIEW

File: .github/README_es.md [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+        <img height="20px" src="https://cdnjs.cloudflare.com/ajax/libs/flag-icon-css/3.4.6/flags/4x3/br.svg">
+    </a>
+    <a href="https://github.com/Shpota/goxygen/tree/master/.github/README_by.md">
+        <img height="20px" src="https://cdnjs.cloudflare.com/ajax/libs/flag-icon-css/3.4.6/flags/4x3/by.svg">
+    </a>
+    <a href="https://github.com/Shpota/goxygen/tree/master/.github/README_fr.md">
+        <img height="20px" src="https://cdnjs.cloudflare.com/ajax/libs/flag-icon-css/3.4.6/flags/4x3/fr.svg">
+    </a>
+    <a href="https://github.com/Shpota/goxygen/tree/master/.github/README_es.md">
+        <img height="20px" src="https://cdnjs.cloudflare.com/ajax/libs/flag-icon-css/3.4.6/flags/4x3/es.svg">
```
Comment (Shpota) [inline .github/README_es.md]: Note, Spanish flag should be larger (25px) when the user browses the Spanish version.
```suggestion
        <img height="25px" src="https://cdnjs.cloudflare.com/ajax/libs/flag-icon-css/3.4.6/flags/4x3/es.svg">
```

File: .github/README_es.md [hunk 2]
```diff
@@ -0,0 +1,134 @@
+<h1 align="center">
+    <a href="https://github.com/Shpota/goxygen/tree/master/.github/README.md">
+        <img height="25px" src="https://cdnjs.cloudflare.com/ajax/libs/flag-icon-css/3.4.6/flags/4x3/gb.svg">
```
Comment (Shpota) [inline .github/README_es.md]:
```suggestion
        <img height="20px" src="https://cdnjs.cloudflare.com/ajax/libs/flag-icon-css/3.4.6/flags/4x3/gb.svg">
```

File: .github/README_es.md [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+
+## Estructura de un proyecto generado (Aplicacion basada en React)
+
+
+    my-app
+    ├── server                   # archivos de GO
+    │   ├── db                   # comunicaciones de MongoDB
+    │   ├── model                # objetos de dominio
+    │   ├── web                  # REST APIs, web server
+    │   ├── server.go            # l punto de entrada del servidor
```
Comment (shpotainna) [inline .github/README_es.md]:
```suggestion
    │   ├── server.go            # el punto de entrada del servidor
```

File: .github/README_es.md [hunk 4]
```diff
@@ -0,0 +1,134 @@
...
+dependencias innecesarias, tan solo utiliza
+[mongo-go-driver](https://github.com/mongodb/mongo-go-driver) en el backend y
+[axios](https://github.com/axios/axios) para pryectos de React y Vue.
+Proyectos en Angular solo utilizan librerias especificas.
+
+## Como contribuir
+
+Si encuentras algun bug o tiene ideas para mejorar el proyecto
+[abre una hilo](https://github.com/Shpota/goxygen/issues) y lo
+solucionaremos lo antes posible. tambien puedes proponer cambios
```
Comment (shpotainna) [inline .github/README_es.md]:
```suggestion
solucionaremos lo antes posible. Tambien puedes proponer cambios
```

**Pull request merged**